package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "fp-group-2",
+  "version": "1.0.0",
+  "description": "## BeHelp",
+  "main": "index.js",
+  "dependencies": {
+    "bcryptjs": "^2.4.3",
+    "cors": "^2.8.5",
+    "dotenv": "^10.0.0",
+    "express": "^4.17.1",
+    "jsonwebtoken": "^8.5.1",
+    "mongodb": "^4.2.1",
+    "mongoose": "^6.1.1",
+    "pg": "^8.7.1",
+    "sqlite3": "^5.0.2"
```

Comment:
we don't actually use mongodb, postgres and sqlite. only mongoose, I think

Reply:
i removed them ;)

Reply:
thank you